client/src/pages/workspace/addBoard/AddBoard.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {IBoardCreation} from "../../../model/Board";
+import {AddBoardProp} from "./props";
+import Button from "@mui/material/Button";
+import "./styles.css";
+import { TextField } from '@mui/material';
+
+const AddBoard: React.FC<AddBoardProp> = ({addBoard}: AddBoardProp) => {
+    const [board, setBoard] = useState<IBoardCreation>({name: '', description: ''})
+    const [isValid, setIsValid] = useState<boolean>(false)
+
+    useEffect(() => {
+        validateBoard(board);
+    }, [board]);
+
+    const addNewBoard = () => {
+        addBoard(board);
+        setBoard({name: '', description: ''});
+    }
+
+    const validateBoard = (board: IBoardCreation) => {
+        if (board.name !== "" && board.description !== "") {
+            setIsValid(true)
+        } else {
+            setIsValid(false)
+        }
+    }
+
+    return (
+        <div className={"board"}>
+            <TextField
+                inputProps={{className: "textFieldInputStyle"}}
```

Comment:
I mean in general, instead of inputProps why not to use className?

Reply:
i dont know how to make it in css